### ISSUE TITLE
Listings - Manual Close

### ISSUE THREAD

**Issue by kathyccheng** [bloom-housing/bloom#1442]
As a leasing agent, 
I might want to close a listing manually
So that I have flexibility in case there is no application due date or something happens 

Notes
* When a leasing agent closes a listing manually, then it should moved to the Closed state. It should over-ride application due date

**Comment by kathyccheng** [comment]
@slowbot I think we should support a manual close? For First Come First Serve listings with no due date, otherwise there's no way to close it. But maybe we want an are you sure alert? What do you think?

**Comment by slowbot** [comment]
@kathyccheng  I think we can use the same modal we use for delete 

![image](https://user-images.githubusercontent.com/13915/124192192-bf480c00-da79-11eb-8e78-af47d0c926ed.png)

**Comment by kathyccheng** [comment]
@dominikx96 the Close action is working on the partners app, it marks the listing as closed. But I don't think it's translating to the public app quite right. The listing that I closed still appears open on the public app. I waited a long time for the cache and refreshed a lot so I don't think that's the issue, and it worked fine for the unpublish. I wonder if we need to change the application due date when we close it, or update the public side to use status vs a due date to filter? cc @seanmalbert @pbn4
